src/oaklib/implementations/ols/__init__.py

```diff
@@ -0,0 +1,3 @@
+from oaklib.datamodels.search import SearchConfiguration
+
+SEARCH_CONFIG = SearchConfiguration()
```

Comment:
this is pretty sketchy, why are there several instances of this objects in different static modules? are these mutable? if so, they shouldn't be the default values for functions regardless of if they live in a module or are constructed in the specification of a function